src/events.rs

```diff
@@ -101,7 +101,10 @@ pub enum WindowEvent {
 pub enum DeviceEvent {
     Added,
     Removed,
+    /// Mouse devices yield `Motion` events where axis `0` is horizontal and axis `1` is vertical.
+    /// Such events will be sent even if the mouse is in a corner of the screen.
```

Comment:
Could we keep the information that a positive value means a movement of the mouse to the bottom or the right, depending on axis?